Package.swift

```diff
@@ -15,6 +15,7 @@ let package = Package(
     .library(name: "HtmlPrettyPrint", targets: ["HtmlPrettyPrint"]),
     .library(name: "HttpPipeline", targets: ["HttpPipeline"]),
     .library(name: "HttpPipelineHtmlSupport", targets: ["HttpPipelineHtmlSupport"]),
+    .library(name: "HttpPipelineTestSupport", targets: ["HttpPipelineTestSupport"]),
```

Comment:
adds snapshot support to `Response`.

it would be fun to be able to snapshot middleware by providing some middleware and a connection and it prints out the transition states to the snapshot too.